spdx-utils/src/test/kotlin/SpdxExpressionTest.kt

```diff
@@ -273,27 +274,24 @@ class SpdxExpressionTest : WordSpec() {
             fun String.decompose() = SpdxExpression.parse(this).decompose().map { it.toString() }
 
             "not split-up compound expressions with a WITH operator" {
-                "GPL-2.0-or-later WITH Classpath-exception-2.0".decompose() shouldContainExactlyInAnyOrder listOf(
+                "GPL-2.0-or-later WITH Classpath-exception-2.0".decompose() should containExactlyInAnyOrder(
                     "GPL-2.0-or-later WITH Classpath-exception-2.0"
                 )
             }
 
             "split-up compound expressions with AND or OR operator but not ones with WITH operator" {
-                "GPL-2.0-or-later WITH Classpath-exception-2.0 AND MIT"
-                    .decompose() shouldContainExactlyInAnyOrder
-                        listOf(
-                            "GPL-2.0-or-later WITH Classpath-exception-2.0",
-                            "MIT"
-                        )
+                "GPL-2.0-or-later WITH Classpath-exception-2.0 AND MIT".decompose() should containExactlyInAnyOrder(
+                    "GPL-2.0-or-later WITH Classpath-exception-2.0",
+                    "MIT"
+                )
             }
 
             "work with LicenseRef-* identifiers" {
                 "LicenseRef-gpl-2.0-custom WITH Classpath-exception-2.0 AND LicenseRef-scancode-commercial-license"
-                    .decompose() shouldContainExactlyInAnyOrder
-                        listOf(
-                            "LicenseRef-gpl-2.0-custom WITH Classpath-exception-2.0",
-                            "LicenseRef-scancode-commercial-license"
-                        )
+                    .decompose() should containExactlyInAnyOrder(
+                    "LicenseRef-gpl-2.0-custom WITH Classpath-exception-2.0",
+                    "LicenseRef-scancode-commercial-license"
```

Comment:
Wrong indentation.

Reply:
Hm, `as-is` formatting is what I get from the IDE. Do you have any hint?

Reply:
Odd, but I can confirm the IDE to *not* complain about the current formatting. However, IMO it should be

                    .decompose() should containExactlyInAnyOrder(
                        "LicenseRef-gpl-2.0-custom WITH Classpath-exception-2.0",
                        "LicenseRef-scancode-commercial-license"
                    )

Reply:
If I do that and apply auto-formatting afterwards the IDE undoes the change. Anyhow, let me fix it up as your proposed formatting looks better.

Reply:
Done.